### DISCUSSION TITLE
proxyState is undefined

### DISCUSSION THREAD

**Discussion by zlever01** [chakra-ui/ark#1184] [Q&A] [answered]
In Nextjs, when installing the ark-ui package I get this error: 

```
- error node_modules/@zag-js/react/node_modules/@zag-js/store/dist/proxy.mjs (234:0) @ snapshot
- error TypeError: proxyState is not iterable

```

When uninstalling ark-ui the error goes away. Any ideas on whats going on?

**Comment by cschroeter** [comment, marked as answer]
I can not reproduce the issue on my Next.js playground. Please be so kind and provide a codesandbox or link a git repoistor and open an issue

**Comment by juliankoehn** [comment]
i had the same error with `"@ark-ui/react": "^0.13.1",` & `0.14.0` in a vite project (npm)

i solved it by adding

```
 "overrides": {
    "@zag-js/store": "^0.16.0"
  }
```

to my package json


> https://github.com/chakra-ui/zag/blob/e4d78be47b4b46e97be943b78561213b022c692c/website/data/overview/faq.mdx#L68
> ## Why am I getting `proxyState is not iterable` error?
>
>This happens when there's a mismatch in the versions of `@zag-js/*` modules or
>multiple instances of `@zag-js/store` are installed. It is common with PNPM
>monorepo users.
>
>To fix this, you can use the `overrides` feature in your `package.json` to
>ensure that all `@zag-js/*` modules are on the same version.
>
>```json
>{
>  "pnpm": {
>    "overrides": {
>     "@zag-js/store": "^0.12.0"
>   }
>  }
>}
```
